### PR TITLE
Trip emote

### PR DESCRIPTION
**Pull request by Darkstick**
### Intent of your Pull Request

By saying *trip, you will now fall over! It does the same effects as being slipped.
With this exciting new addition, you will now be able to:

- Do more RP! (please)
- Replicate all sorts of cute shojo scenes!
- Have a cool way to stop chasing someone without it being awkward!
- Faceplant!
- Look clumsy!
- Smash your face!
- Frame the clown for slipping you and watch as the rest of the security team beats them!

And so much more!
Tie your shoes together and hurt yourself today!

#### Changelog

:cl:  
rscadd: Trip emote
/:cl:

### CODE REVIEW

**Review by ReddicusDragon — CHANGES_REQUESTED** [review]
"trips and fall!" should be "trips and falls!"

**Comment by monster860** [comment]
What would be nice is a way to do this without having to stop, press t, and type *trip. Kinda ruins the point of tripping while chasing someone

**Comment by athena413** [comment]
neat

**Review by Hopekz — APPROVED** [review]

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -111,6 +111,17 @@
 		var/mob/living/L = user
 		L.SetSleeping(200)
 
+/datum/emote/living/trip
+	key = "trip"
+	key_third_person = "trip"
```
Comment (alexkar598) [inline code/modules/mob/living/emote.dm]:
```suggestion
	key_third_person = "trips"
```

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Review by alexkar598 — APPROVED** [review]

**Pull request merged**